src/verify.c

```diff
@@ -44,18 +44,21 @@ static int consttime_equal(const unsigned char *x, const unsigned char *y) {
     return !r;
 }
 
-int ed25519_verify(const unsigned char *signature, const unsigned char *message, size_t message_len, const unsigned char *public_key) {
+int ed25519_verify(const unsigned char *signature, const unsigned char *message, size_t message_len, const unsigned char *public_key)
```

Comment:
I don't have opinions on stylistic changes such as this one, that's entirely up to the maintainer, but I would strongly recommend to not put them in the same commit as actual semantic changes.